app/javascript/components/Components/Header.js

```diff
@@ -8,41 +8,60 @@ class Header extends Component {
       current_user,
       new_user_route,
       sign_in_route,
-      sign_out_route
-    } = this.props
-    console.log("logged_in:", logged_in)
-    console.log("current_user:", current_user)
+      sign_out_route,
+    } = this.props;
+    console.log("logged_in:", logged_in);
+    console.log("current_user:", current_user);
 
     return (
       <>
         <Nav>
           <NavItem>
-            <a href={'/'} className="nav-link">Path Unknown</a>
+            <a href={"/"} className="nav-link">
```

Comment:
All internal routes (routes being defined by react-router) should use `NavLink to=`